src/iexamine.cpp

```diff
@@ -621,8 +616,18 @@ void iexamine::cardreader(player *p, map *m, const tripoint &examp)
             }
         }
         add_msg(_("You insert your ID card."));
-        add_msg(m_good, _("The nearby doors slide into the floor."));
         p->use_amount(card_type, 1);
+
+        bool open = 0;
```

Comment:
It's a `bool`, why are you assigning an `int`? Same below with `1`, why the `int`? It should be `true` / `false`.